### PR TITLE
IsAirgap, Sequence, Cursor, ChannelName, VersionLabel, ReleaseNotes

### PR DESCRIPTION
**Pull request by laverya**

### CODE REVIEW

File: kotsadm/pkg/handlers/config.go [hunk 1]
```diff
@@ -200,7 +200,16 @@ func LiveAppConfig(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
-	renderedConfig, err := kotsconfig.TemplateConfigObjects(kotsKinds.Config, configValues, appLicense, template.LocalRegistry{})
+	versionInfo := template.VersionInfo{
```
Comment (emosbaugh) [inline kotsadm/pkg/handlers/config.go:203]: could you make a function to construct this struct?

File: kotsadm/pkg/preflight/preflight.go [hunk 2]
```diff
@@ -54,7 +54,7 @@ func Run(appID string, sequence int64, archiveDir string) error {
 			return errors.Wrap(err, "failed to get registry settings for app")
 		}
 
-		renderedPreflight, err := render.RenderFile(renderedKotsKinds, registrySettings, []byte(renderedMarshalledPreflights))
+		renderedPreflight, err := render.RenderFile(renderedKotsKinds, registrySettings, sequence, isAirgap, []byte(renderedMarshalledPreflights)) // TODO fix
```
Comment (emosbaugh) [inline kotsadm/pkg/preflight/preflight.go:57]: fix what?

File: kotsadm/pkg/render/render.go [hunk 3]
```diff
@@ -132,6 +141,7 @@ func RenderDir(archiveDir string, a *apptypes.App, downstreams []downstreamtypes
 		IsAirgap:         a.IsAirgap,
 		AppSlug:          a.Slug,
 		IsGitOps:         a.IsGitOps,
+		AppSequence:      a.CurrentSequence + 1,
```
Comment (emosbaugh) [inline kotsadm/pkg/render/render.go:144]: why +1? maybe justify this in a comment

File: kotsadm/pkg/handlers/config.go [hunk 4]
```diff
@@ -200,7 +200,16 @@ func LiveAppConfig(w http.ResponseWriter, r *http.Request) {
...
+		Sequence:     liveAppConfigRequest.Sequence + 1,
```
Comment (emosbaugh) [inline kotsadm/pkg/handlers/config.go:204]: why +1? maybe justify this in a comment

**Review by emosbaugh — APPROVED** [review]

File: kotsadm/pkg/preflight/preflight.go (hunk 2 above)
Comment (laverya) [reply kotsadm/pkg/preflight/preflight.go:57]: whoops, left in the todo
will remove

File: kotsadm/pkg/render/render.go (hunk 3 above)
Comment (laverya) [reply kotsadm/pkg/render/render.go:144]: +1 because it provides the most recent sequence, and we want the next sequence here

**Review by emosbaugh — APPROVED** [review]

**Comment by laverya** [comment]
holding off on merging this until 1.19 is out&passes regression

**Pull request merged**